_inc/client/state/period-toggle/reducer.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { SET_PLAN_DURATION } from 'state/action-types';
+
+const planDuration = ( initialState = 'yearly', action ) => {
+	switch ( action.type ) {
+		case SET_PLAN_DURATION:
+			return action.duration;
+
+		default:
+			return initialState;
+	}
+};
+
+export const reducer = combineReducers( {
+	planDuration,
```

Comment:
How about using a more explicit key?

```suggestion
	selectedPlanDuration,
```

Reply:
Sounds good. Updated with your suggestion.